NEWS.md

```diff
@@ -1,6 +1,7 @@
 # babynames 0.3.0.9000
 
 * Add data validity/regression tests (@russellpierce, #27)
+* Add data for 2016 and 2017. Minor changes to data from previous years to match external data sources. (@russellpierce #30)
```

Comment:
Can you please add a new line before the bullet, and a comma before the issue number?

Reply:
Done.